tests/wpwebdriveracceptance/DatePicker3Cept.php

```diff
@@ -0,0 +1,175 @@
+
+<?php
+
+$I = new WpwebdriveracceptanceTester($scenario);
+$I->wantTo('Test the datepicker cases');
+
+//Login to backend
+$I->amOnPage("/wp-login.php");
+$I->fillField('Username', 'test_user');
+$I->fillField('Password','test_user');
+//$I->fillField("#user_login", "admin");
+//$I->fillField("#user_pass", "admin");
+$I->click('Log In');
+
+$I->wait(5);
+
+$output = '';
+
+$result = '';
+
+$array = array("2016-01-15","1/15/2016","01/15/2016","15/1/2016","15/01/2016","1-15-2016","01-15-2016","15-1-2016","15-01-2016");
+$i = 0;
+
+foreach($array as $ar){
+	$option = $ar;
+
+	//Goto the Event Setting tab and click Display tab
+	$I->amOnPage('/wp-admin/edit.php?page=tribe-common&tab=display&post_type=tribe_events');
+	$I->waitForElement(['name' => 'datepickerFormat'], 5); // secs
+	$I->scrollTo(["css" => "#tribe-field-weekDayFormat"]);
+	$jQueryDatepicker = 'jQuery(\'select[name="datepickerFormat"]\').val(' . $i  . ').prop("selected","selected").change();';
+	$I->executeJS($jQueryDatepicker);
+	$I->wait(2);
+	$I->click('Save Changes');
+
+	//Increment $i
+	$i = $i + 1;
+
+	//Move to Frontend event date
+	$I->amOnPage('/events/today/');
+
+	$jQueryDay = 'jQuery(\'#tribe-bar-date\').val()';
+
+	$frontdate = $I->executeJS('return jQuery("#tribe-bar-date").val()');
+	
+	$I->wait(2);
+
+	//For Date Format Y-m-d
+	if(validateDate($option, 'Y-m-d') == 1){
```

Comment:
It looks as thouhg this and the next 8 if blocks essentially do the same thing - only the date format changes. How about we put the different formats in an array and then use a single foreach block instead? It could really shorten the length of this file and, if we want to change something, we'd change it once instead of in 9 separate locations.